quint/src/generated/Quint.g4

```diff
@@ -93,36 +97,36 @@ row : | (IDENTIFIER ':' type ',')* ((IDENTIFIER ':' type) (',' | '|' (IDENTIFIER
 // check the precedence table in Java:
 // https://www.cs.bilkent.edu.tr/~guvenir/courses/CS101/op_precedence.html
 expr:           // apply a built-in operator via the dot notation
-                expr '.' nameAfterDot (LPAREN argList? RPAREN)?     # dotCall
+                expr '.' nameAfterDot argList?                      # dotCall
         |       lambda                                              # lambdaCons
                 // Call a user-defined operator or a built-in operator.
                 // The operator has at least one argument (otherwise, it's a 'val').
-        |       normalCallName '(' argList? ')'                     # operApp
+        |       normalCallName argList                              # operApp
```

Comment:
This is the only change in the grammar which I can easily see as a simplification. Could you explain a bit the motive for the other changes?